src/main/java/qlik/jdbc/connector/ConnectorImpl.java

```diff
@@ -4,92 +4,148 @@
 import io.grpc.*;
 import io.grpc.stub.StreamObserver;
 import qlik.connect.ConnectorGrpc;
+import qlik.connect.GrpcServer;
 import qlik.connect.GrpcServer.*;

```

Comment:
Is this import redundant now?

Reply:
We need both. One is for the GrpcServer class and the other one for all the subclasses under GrpcServer